build/common.sh

```diff
@@ -518,21 +497,14 @@ function kube::build::build_image() {
   kube::version::save_version_vars "${build_context_dir}/kube-version-defs"
 
   cp build/build-image/Dockerfile ${build_context_dir}/Dockerfile
-  kube::build::update_dockerfile
 
-  # We don't want to force-pull this image because it's based on a local image
-  # (see kube::build::build_image_cross), not upstream.
-  kube::build::docker_build "${KUBE_BUILD_IMAGE}" "${build_context_dir}" 'false'
-}
-
-# Build the kubernetes golang cross base image.
-function kube::build::build_image_cross() {
-  kube::build::ensure_golang
+  if kube::build::is_osx; then
+    sed -i "" "s/KUBE_BUILD_IMAGE_CROSS_TAG/${KUBE_BUILD_IMAGE_CROSS_TAG}/" ${build_context_dir}/Dockerfile
+  else
+    sed -i "s/KUBE_BUILD_IMAGE_CROSS_TAG/${KUBE_BUILD_IMAGE_CROSS_TAG}/" ${build_context_dir}/Dockerfile
+  fi
 
-  local -r build_context_dir="${LOCAL_OUTPUT_ROOT}/images/${KUBE_BUILD_IMAGE}/cross"
-  mkdir -p "${build_context_dir}"
-  cp build/build-image/cross/Dockerfile ${build_context_dir}/Dockerfile
-  kube::build::docker_build "${KUBE_BUILD_IMAGE_CROSS}" "${build_context_dir}"
+  kube::build::docker_build "${KUBE_BUILD_IMAGE}" "${build_context_dir}" 'false'
```

Comment:
I'm not sure... I can force pull to be safe. I'd like to say that we should just be disciplined and tags should be immutable once pushed. It will improve the speed of quick-release to skip it though.